core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/DataSource.java

```diff
@@ -32,11 +34,11 @@
 public @interface DataSource {
 
     /**
-     * When set to a non-blank string, stores as the {@code sling:resourceType} attribute of a {@code datasource node}
-     * of the current widget
-     * @return String value
+     * Defines the path to the component in JCR
```

Comment:
not sure that a datasource can be called a component. perhaps we can change it to something like this?
```suggestion
     * Defines the {@code sling:resourceType} attribute of a {@code datasource node} of the current widget
```